spec/serializers/api/v3/ecf/participant_serializer_spec.rb

```diff
@@ -53,6 +53,7 @@ module ECF
                       withdrawal: nil,
                       deferral: nil,
                       created_at: ect_profile.created_at.rfc3339,
+                      induction_end_date: ect_profile.induction_completion_date&.strftime("%Y-%m-%d"),
```

Comment:
can we change the tests to explicitly test the value returned as in "2022-01-12" or `nil` below?